documentation/concepts/check.md

```diff
@@ -4,25 +4,88 @@ menu: check
 
-## Flags
-The command uses a flag to indicate the kind of report to generate:
+Before performing a deployment to the target database (most notably, production), you might want to look over what you’re able to do and understand one or more of the following: 
 
-### `-changes` 
+- Does this set of changes affect the objects I expect it to, or will I be inadvertently having an impact of something else? 
+- What database changes have been made recently, that coincide with the changes in database performance, we are seeing? Are the two related? 
+- Is the production database in the same state you were expecting when you began developing your changes? Has anything about the target database changed that would mean my changes no longer have the desired effect?  
+- Does our approach to database change development meet our internal policies? Are our migration scripts adhering to our naming conventions, for example? Are we following the security best-practices required of us by our external regulatory requirements? 
 
-This is used to report on the pending changes that have yet to be applied to the database. You might use this as part of an in-person review process or as a validation of the effect of all pending changes.
-- The command then runs a comparison engine to identify differences between the temporary database at two points:
-    - The state equivalent to the current version of the target database
-    - The state once all pending migrations have been run
+Each of these scenarios can be met with the `check` command, using the corresponding flag: 
+<table class='table'>
+<tr>
+    <td><strong>Scenario</strong></td>
+    <td><strong>Command &amp; Flag</strong></td>
+    <td><strong>Output</strong></td>
+</tr>
+<tr>
+    <td><em>In beta</em><br>Will these changes have the effect I am expecting?</td>
+    <td>check <b>-changes</b></td>
+    <td>ChangeReport.html,<br>ChangeReport.json</td>
+</tr>
+
+<tr>
+    <td><em>In beta</em><br>What database changes have been made recently?</td>
+    <td>check <b>-changes</b></td>
+    <td>ChangeReport.html,<br>ChangeReport.json</td>
+</tr>
+
+<tr>
+    <td><em>Coming soon</em><br>Is the production database in the state I am expecting it to be in?</td>
+    <td>check <b>-drift</b></td>
+    <td>&nbsp;</td>
+</tr>
+<tr>
+    <td><em>Coming soon</em><br>Are our changes following internal policies?</td>
+    <td>check <b>-drift</b></td>
+    <td>&nbsp;</td>
+</tr>
+</table>
+
+## `Check –changes` 
+### Overview 
+the `–changes` flag produces a report indicating differences between your target database (ie. the one you want to change) and what your migration scripts define (ie. the set of instructions you want to use to change your target database).  
+You can use this capability pre- and post-deployment: 
+- In pre-deployment scenarios to check the effect of your pending changes 
+- In post-deployment scenarios to capture a history of changes for retrospective auditing or reporting 
```

Comment:
Do we need to mention the post-deployment scenario is something that is coming soon? It's not currently a capability afaik

Reply:
Not sure if it is intended as a feature of the tool or a feature of the wider process - if you grabbed the artifacts before deployment the you could use them post-deployment to report on what you did. I'll ask ...